indra/assemblers/pybel_assembler.py

```diff
@@ -161,6 +163,19 @@ def _assemble_active_form(self, stmt):
         for edge_data in edge_data_list:
             self.model.add_edge(subj_node, obj_node, attr_dict=edge_data)
 
+    def _assemble_complex(self, stmt):
+        members_list = []
+        for member in stmt.agent_list():
+            func, namespace, name = _get_agent_grounding(member)
+            members_list.append({
```

Comment:
I would make this recursive for getting the JSON of any given member

Reply:
So I was wondering about that--can members of a complex be anything? e.g., can they be modified proteins? Other complexes?